docs/spec/distribution/transactions.md

```diff
@@ -1,16 +1,15 @@
 # Transactions
 
-## TxWithdrawDelegationRewardsAll
+## MsgWithdrawDelegationRewardsAll
 
 When a delegator wishes to withdraw their rewards it must send
-`TxWithdrawDelegationRewardsAll`. Note that parts of this transaction logic are also
+`MsgWithdrawDelegationRewardsAll`. Note that parts of this transaction logic are also
 triggered each with any change in individual delegations, such as an unbond,
 redelegation, or delegation of additional tokens to a specific validator.  
 
 ```golang
-type TxWithdrawDelegationRewardsAll struct {
-    delegatorAddr sdk.AccAddress
-    withdrawAddr  sdk.AccAddress // address to make the withdrawal to
+type MsgWithdrawDelegationRewardsAll struct {
+    DelegatorAddr sdk.AccAddress
```

Comment:
Why did we decide to change this logic and no longer permit a withdrawal address to be specified per-withdrawal?

Reply:
Because of automatic withdrawals which need to occur during power changes etc. easy just to keep a record